cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -83,7 +83,7 @@ func (s *server) Start(_ context.Context, host component.Host) error {
 		return err
 	}
 
-	tm := tenancy.NewManager(&s.config.Tenancy)
+	tm := tenancy.NewManager(&s.config.QueryOptionsBase.Tenancy)
```

Comment:
since the struct is embedded, I don't think you need `QueryOptionsBase.` qualifier